common/src/main/cpp/control/PIDController.cpp

```diff
@@ -46,20 +46,25 @@ void PIDController::SetSetpoint(double setpoint) {
   _setpoint = setpoint;
 }
 
+void PIDController::SetIZone(double threshold) {
+  _threshold = threshold;
+}
+
 double PIDController::GetSetpoint() {
   return _setpoint;
 }
 
 void PIDController::SetWrap(double range) {
-  _wrap_range = range;
+  _wrapRange = range;
 }
 
-double PIDController::Calculate(double processVariable, double dt) {
+double PIDController::Calculate(double processVariable, double dt, double feedforward = 0.0) {
```

Comment:
You can remove `= 0.0` since it's already been declared in the header. We don't double-declare defaults